src/vtp/core/common.py

```diff
@@ -140,20 +140,80 @@ def verify_election_data_dir(election_data_dir: str):
             )
 
     @staticmethod
-    def get_guid_dir(guid: str) -> str:
-        """Return the default runtime location for a guid based workspace"""
+    def get_generic_ro_edf_dir() -> str:
+        """
+        Will return a generic EDF workspace so to be able to execute
+        generic/readonly commands.  It is 'readonly' because any
+        number of processes could be executing in this one git
+        workspace at the same time and if any them wrote anything, it
+        would be bad.
+        """
+        edf_path = os.path.join(
+            Globals.get("DEFAULT_RUNTIME_LOCATION"),
+            Globals.get("MOCK_CLIENT_DIRNAME"),
+            "scanner.00",
+        )
+        # Need to verify that there is only _one_ directory in the edf_path
+        dirs = [
+            name
+            for name in os.listdir(edf_path)
+            if os.path.isdir(os.path.join(edf_path, name))
+        ]
+        if len(dirs) > 1:
+            raise ValueError(
+                f"The mock client directory ({edf_path}) ",
+                "contains multiple subdirs - there can only be one ",
+                "as there should only be one EDF clone in this directory",
+            )
+        if not dirs:
```

Comment:
Accepted with two modifications to the above: 1) it should be an '==' test and 2) since the preceeding if clause results in an exception raise, black wants the elif to be an if.  Assuming both are ok.